sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -3028,6 +3032,15 @@ case class ConvertTimezone(
     sourceTs: Expression)
   extends TernaryExpression with ImplicitCastInputTypes with NullIntolerant {
 
+  def this(targetTz: Expression, sourceTs: Expression) = {
+    this(
+      Literal(SQLConf.get.sessionLocalTimeZone),
```

Comment:
Please, make the expression as time zone aware via extending `TimeZoneAwareExpression`.